contracts/JBController.sol

```diff
@@ -475,28 +508,31 @@ contract JBController is IJBController, JBTerminalUtility, JBOperatable, Reentra
     returns (uint256 beneficiaryTokenCount)
   {
     // Can't send to the zero address.
-    require(_reservedRate == 10000 || _beneficiary != address(0), '0x2f: ZERO_ADDRESS');
+    if (_reservedRate != MAX_TOKEN_RATE && _beneficiary == address(0)) {
+      revert BENEFICIARY_ZERO_ADDRESS();
+    }
```

Comment:
I think we should check the reserve rate separately, no? != MAX_TOKEN_RATE also probably isn't right – the separate if should be if (_reservedRate > MAX_TOKEN_RATE)

cc: @mejango

Reply:
I'd say : 
`if(beneficiary == address(0) && _reservedRate < 10000)`
If the reserved rate is 100, it can be the 0 address
 + mini gas optim, testing first the address (address(0) which should be less frequently true, avoiding reserveRate<10000 test most of the time